plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java

```diff
@@ -395,7 +395,7 @@ public List<CassandraPartition> getPartitions(CassandraTable table, List<Set<Obj
             rows = queryPartitionKeysLegacyWithMultipleQueries(table, filterPrefixes);
         }
 
-        if (rows == null) {
+        if (rows.isEmpty()) {
             // just split the whole partition range
             return ImmutableList.of(CassandraPartition.UNPARTITIONED);
```

Comment:
This used to be a dead code. 
So the correct "naive update" would be to remove the block.
Is this codepath exercised? 

cc @ebyhr

Reply:
I think it's a dead code. Let's remove that.

Reply:
Thanks for your advice, I updated the commits.

Reply:
There is a failing check, but it's not caused by this PR and can be ignored.